src/utils.h

```diff
@@ -36,6 +36,7 @@ SPDX-License-Identifier: GPL-2.0-or-later
 
 int preg_replace(const char *re, char *replacement, char *subject, char *result, size_t pcre2_result_len);
 int split(char* input_string, char** output_arr, char* delim);
+int scandir_filter_no_dot(const struct dirent *);
```

Comment:
Same consideration about leaving vars names in declarations.

Reply:
I will change it to add the var names.